src/app/public/src/modules/nav/nav.service.ts

```diff
@@ -46,14 +50,59 @@ export class StacheNavService {
     return /^(https?|mailto|ftp):+|^(www)/.test(path);
   }
 
+  public updateRoutesOnScroll(routes: StacheNavLink[]) {
+    if (routes && routes.length) {
+      this.updateView(routes);
+    }
+  }
+
+  // Updates ng-class level trigger to show/hide blue border marking location on page
+  public updateView(routes: StacheNavLink[]) {
+    this.trackPageOffset();
+    this.isCurrent(routes);
+  }
+
+  // Checks to see if stache-wrapper exists and provides offset from top of window
+  private getHeaderHeight() {
+    const stacheWrapper = this.windowRef.nativeWindow.document.querySelector('.stache-wrapper');
```

Comment:
What happens if there is not Stache Wrapper, and they try to use the TOC stand alone?

Reply:
Currently the ToC isn't set up to be used outside of the context of the wrapper layouts. That's not something we took in to consideration when initially building the component, so I think it's out of scope for what this PR is supposed to be fixing. I think we'd probably want to consider this when/if we decide to break it out. Or at least think about handling this at a later time, otherwise.